langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -266,7 +266,7 @@ public isolated function fromJsonStringWithType(string str, typedesc<anydata> t
     name: "fromJsonStringWithType"
 } external;
 
-# Merges two json values.
+# Merges two `json` values.
```

Comment:
```suggestion
# Merges two JSON values.
```

Reply:
@praneesha here what I wanted to say is actually, values of ballerina type `json`. Do you think this is ok to replace it with JSON, because in most of the places we have used `int` and '`float`

Reply:
The +1 to use backticks.